work9/index.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+const ul = document.getElementById("js-list");
+const data = [{
+  to: "bookmark.html",
+  img: "1.png",
+  alt: "画像1",
+  text: "ブックマーク",
+},
+{
+  to: "message.html",
+  img: "2.png",
+  alt: "画像2",
+  text: "メッセージ"
+},
+]
+
+function getData() {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve(data);
+
+      reject(new Error("エラー"));
+
+    }, 3000);
+  })
+}
+
+async function createElement() {
+  const fragment = document.createDocumentFragment();
+
+  const getObj = await getData();
+
+  ul.style.backgroundImage = "none";
+
+  getObj.forEach((item) => {
+    const li = document.createElement("li");
+    const a = document.createElement("a");
+    const img = document.createElement("img");
+
+    a.href = item.to;
+    a.textContent = item.text;
+    img.src = item.img;
+    img.alt = item.alt;
+
+    fragment.appendChild(li).appendChild(a).prepend(img);
+  });
+  ul.appendChild(fragment);
+}
+
+
+function loading() {
+  ul.style.backgroundImage = "url(./img/loading-circle.gif)";
+  ul.style.backgroundRepeat = "no-repeat";
+  ul.style.height = "100px";
+}
+
+loading();
+
+createElement();
```

Comment:
I ask you, What is the job of createElement function?

Reply:
> What is the job of createElement function?


Does this mean that the createElement function is no longer a function that creates an element?
If so, I've divided it into separate functions, so please check.

f53ee5b

Reply:
onece again.

I ask you, What is the job of createElement function?

Reply:
> I ask you, What is the job of createElement function?
please answer my question.

"The createElement function creates a list element and treats it as a child element of the ul element"

Reply:
https://github.com/yasu-pro/js-handson/pull/9#discussion_r688514232

Same as above
It is pointed out that the function name and the content processed in it are different.

It will be the commit number.
Please check.

94ce008